src/Transform/ONNX/ShapeInferencePass.cpp

```diff
@@ -97,8 +97,9 @@ class ShapeInferencePass
   }
 
   static LogicalResult runShapeInferenceOnRegion(Region &r) {
-    std::function<void(Region &)> doShapeInference =
-        &ShapeInferencePass::runShapeInferenceOnRegion;
+    std::function<void(Region &)> doShapeInference = [&](Region &region) {
```

Comment:
nit: can you omit the `&` in `[&]`? it doesn't seem like anything is captured

Reply:
if applicable, make sense to remove the captured context